src/main/java/app/bpartners/geojobs/service/detection/DetectionCreationMapper.java

```diff
@@ -77,6 +83,28 @@ public Detection apply(
         .build();
   }
 
+  private List<DetectableObjectConfiguration> getDetectableObjectConfigurations(
+      String detectionId, CreateDetection createDetection) {
+    List<DetectableObjectConfiguration> detectableObjectConfigurations;
+    if (createDetection.getDetectableObjectModelList() == null
+        || createDetection.getDetectableObjectModelList().isEmpty()) {
+      detectableObjectConfigurations =
+          detectableObjectTypeMapper.mapDefaultConfigurationsFromModel(
+              detectionId, createDetection.getDetectableObjectModel().getModelName());
```

Comment:
Do you still need to check if `createDetection.getDetectableObjectModel()` here or is it done earlier ? Because it may produce NullPointerException (NPE)